sdccc/src/test/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMultiStateTestTest.java

```diff
@@ -275,7 +275,18 @@ public void testRequirementR0097GoodOneAcceptableSequenceSeen() throws Exception
                 null,
                 null,
                 buildOperatorContext(
-                        OPERATOR_CONTEXT_DESCRIPTOR_HANDLE2, OPERATOR_CONTEXT_STATE_HANDLE2, BigInteger.TWO));
+                        OPERATOR_CONTEXT_DESCRIPTOR_HANDLE, OPERATOR_CONTEXT_STATE_HANDLE2, BigInteger.TWO));
```

Comment:
Why the change?

Reply:
this is to make all sequence ids relevant for the test, it was useful when debugging the R0097 test case